component/common/src/main/java/org/exoplatform/platform/common/software/register/service/RestPLFRegistration.java

```
@@ -48,11 +48,11 @@ public Response checkConnection() throws Exception {
             return Response.ok(String.valueOf(HttpURLConnection.HTTP_OK == urlConn.getResponseCode())).build();
         } catch (MalformedURLException e) {
             if(LOG.isErrorEnabled()) {
-                LOG.error("LeadCapture : Error creating HTTP connection to the server : " + pingServerURL, e);
+                LOG.error("LeadCapture : Error creating HTTP connection to the server : " + pingServerURL);
```

I don't think we should swallow exception. The issue here is to not display the stacktrace when community is not reachable. This case throws a UnknownHostException. So I think we should instead catch this particular exception, and in this case adding a warn log "Cannot reach server xxxx" without the exception. In others cases, it is an error, so the exception should not be swallowed.